pkg/rebalancer/storage/controller/service/rebalancer.go

```diff
@@ -148,56 +139,69 @@ func NewRebalancer(opts ...RebalancerOption) (Rebalancer, error) {
 			statefulset.WithOnErrorFunc(func(err error) {
 				log.Error(err)
 			}),
-			statefulset.WithOnReconcileFunc(func(statefulSetList map[string][]statefulset.StatefulSet) {
+			statefulset.WithOnReconcileFunc(func(ctx context.Context, statefulSetList map[string][]statefulset.StatefulSet) {
```

Comment:
I  think what the original comment meant is the divide the  reconcile logic from here.
e.g. 
```
...
statefulset.WithOnReconcileFunc(statefulSetReconcile),
....

func statefulSetReconcile(ctx context.Context, statefulSetList map[string][]statefulset.StatefulSet) {
     ....
}

Reply:
thank you.
I fixed: https://github.com/vdaas/vald/pull/1187/commits/51d34852992605ac47845619e024b2840db5be67